### PR TITLE
Feature/seab 1715/git secrets lambda

### PR DESCRIPTION
**Pull request by Ldcabansay**
Added package.json to install husky and run script for installing git-secrets with a scan. Added test for CircleCI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "lambda",
+  "version": "0.1.0",
+  "license": "Apache License 2.0",
+  "scripts": {
+    "install-git-secrets": "./scripts/install-git-secrets.sh"
```
Comment (coverbeck) [inline package.json:6]: I would separate the install and running of it. Missed this in reviewing Walt's. But not crucial.

**Review by coverbeck — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Ldcabansay) [reply package.json:6]: its more for the ci job and initial install, after you can easily just run git secrets --scan after is been installed. Didn't really see a reason to add an npm script for that if it was just going to call git secrets anyway

**Review by garyluu — APPROVED** [review]

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 # Created by .ignore support plugin (hsz.mobi)
+git-secrets.sh
```
Comment (david4096) [inline .gitignore:2]: You could alternatively just delete it when done in your bash.

**Review by david4096 — APPROVED** [review]
Confirmed this works by trying to add a credentials file. People need to be told to run `npm install`, `npm run install-git-secrets`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+Till we find a better solution, backup and document AWS lambdas
+
+## Installing git-secrets
+Dockstore uses git-secrets to help make sure that keys and private data stay out
+of the source tree.
+To install and check for git secrets:
+
+```
+npm ci
+npm run install-git secrets
```
Comment (david4096) [inline README.md:11]: Thanks

File: scripts/install-git-secrets.sh [hunk 4]
```diff
@@ -10,72 +10,8 @@ set -u
...
-  # return success code (true) since git secrets --add
-  # will return a 1 if the pattern is already set
-  # and that will fail when running
-  # npm run install-git-secrets
-  git secrets --add '[a-fA-F0-9]{30,}' || true
-  #if scan produces output script should error, check for false positives
-  git secrets --scan
-fi
+#grab install script from UI repo
+GIT_SECRETS_SOURCE="https://raw.githubusercontent.com/dockstore/dockstore-ui2/develop/scripts/install-git-secrets.sh"
```
Comment (denis-yuen) [inline scripts/install-git-secrets.sh:14]: Ditto, from other PR.

**Review by denis-yuen — APPROVED** [review]

**Pull request merged**